charts/leonardo/values.yaml

```diff
@@ -115,3 +115,23 @@ zombieMonitorCron:
 vault:
   # vault.pathPrefix -- (string) Vault path prefix for secrets. Required if vault.enabled.
   pathPrefix:
+  # Migration credentials only referenced if migration.enabled == true
+  migration:
+    # vault.migration.path -- Vault path to secret containing DB credentials
+    path: null
+    # vault.migration.dbUsernameKey -- Key in Vault secret to DB username
+    dbUsernameKey: "db_user"
+    # vault.migration.DbPasswordKey -- Key in Vault secret to DB password
+    dbPasswordKey: "db_password"
+
+migration:
+  # migration.enabled -- (bool) Whether to run a Liquibase migration during sync
+  enabled: false
```

Comment:
is this necessary now that `liquibaseCommand` also governs whether the migration will be run?

Reply:
`enabled` controls whether any liquibase will be run at all, while `liquibaseCommand` controls what kind of migration will be run. `enabled` being false is how we can safely add this to the chart, because every environment won't suddenly be trying to create the migration job and secret--just where we explicitly enable it, so dev for now. `liquibaseCommand` is how we can test it--we want the job to be created, but we don't want it to start writing to the database on its first go.

Reply:
I'm just thinking it could be surprising to ppl, when we need to enable the migration, we need to both update `enable` and update `liquibaseCommand`...maybe add a comment for that?